### PR TITLE
User guide

### PR DESCRIPTION
**Pull request by adam23232323**
## 📋 | Pre-PR

Before submitting a pull request make sure you have:

- [ ] Read the [contributing guidelines](https://fit3170-fy-project-7.github.io/RABIT-DOCS/dev-guide/contributing.html).
- [ ] Searched the bug tracker for similar pull requests.

- [ ] This PR is related to the code for RABIT documentation and will be merged to the [RABIT-DOCS](https://github.com/FIT3170-FY-Project-7/RABIT-DOCS) repository.

---

## 📄 | Licensing

> In order to be accepted and merged into RABIT, **each piece of code must be released under [ISC license](https://github.com/FIT3170-FY-Project-7/RABIT-DOCS/blob/main/LICENSE.md) or a compatible license**.
>

*Check **one** of the following options:*

- [ ] I am the original author of this code, and I am willing to release it under the ISC license.
- [ ] I am not the original author of this code, but it is released under the ISC license or a compatible license (please provide evidence).

---

## ℹ️ | Description / Further Information

> Explanation of __the purpose and effect__ of your *pull request* goes here.
>
> Provide __context and examples__ as necessary.

### CODE REVIEW

File: src/user-guide/create-plot.md [hunk 1]
```diff
@@ -1 +1,56 @@
...
+One of RABIT's premier features is the creation of Bayesian inference plots.
+
+
+
+
+## 1. Navigate to the upload page
+
+Navigate via the sidebar or go directly to the '/upload' URL. On the upload page, users will be able to create a plot by filling out the mandatory fields.
+
+![App Screenshot](../images/create-plot/navigation.png)
```
Comment (megascrapper) [inline src/user-guide/create-plot.md]: Rename link to `../images/create-plot/navigation.PNG` or image filename to `navigation.png`

(note: you need to rename all instances)

**Review by megascrapper — CHANGES_REQUESTED** [review]
The screenshots don't appear on Linux because of capitalisation mismatch (i.e. the image filename has `.PNG` extension but the link has `.png` extension)

Thsi could be a problem for the github pages deploy, so you should fix it.

File: src/user-guide/create-plot.md [hunk 2]
```diff
@@ -1 +1,56 @@
...
 # Creating a plot
+
+One of RABIT's premier features is the creation of Bayesian inference plots.
+
+
+
+
+## 1. Navigate to the upload page
+
+Navigate via the sidebar or go directly to the '/upload' URL. On the upload page, users will be able to create a plot by filling out the mandatory fields.
```
Comment (megascrapper) [inline src/user-guide/create-plot.md]: Use code style when referencing URL path i.e. surround with backticks

File: src/user-guide/view-plot.md [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+# Viewing a plot
+
+One of RABIT's premier features is viewing Bayesian inference plots. For a plot to be viewed, it must first be created. If you haven't already done so, read the plot creation guide [here](https://github.com/FIT3170-FY-Project-7/RABIT-DOCS/blob/main/src/user-guide/create-plot.md).
+
+
+
+
+## 1. Navigate to the plots page
+
+Navigate to 'Explore' from the sidebar, or go directly to the '/plots' URL.
```
Comment (megascrapper) [inline src/user-guide/view-plot.md]: Use code style when referencing URL path i.e. surround with backticks

**Comment by adam23232323** [comment]
Changes made and pushed

**Review by megascrapper — APPROVED** [review]

**Pull request merged**